react-native/react/menubar/index.render.desktop.js

```diff
@@ -200,49 +228,84 @@ class FolderList extends Component {
 
     // Remove folders that are just our personal ones, we'll add those in later
     // For consistency. Since we aren't gauranteed we have favorited our own folders.
-    const folders = this.props.folders.filter(f => stripPublicTag(f.folderName) !== username)
-
-    const personalPrivateFolder: FolderInfo = {
-      folderName: username,
-      isPublic: false,
-      isEmpty: true,
-      openFolder: this.props.openKBFSPrivate
+    const folders = this.props.folders.filter(f => f.type === 'entry' || stripPublicTag(f.folderName) !== username)
+
+    let privateFolders = []
+    let publicFolders = []
+
+    const isLoggedIn = true // TODO plumb this through
```

Comment:
eventually we'll allow you to be logged out and see kbfs public folders, for now this is hardcoded